worker-document/pom.xml

```diff
@@ -49,6 +49,10 @@
             <groupId>com.github.cafapi.util</groupId>
             <artifactId>util-moduleloader</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
```

Comment:
@mmaBel By the way, this new dependency is a worry from a breaking change point of view.  It is a dependency that the workers didn't previously have and there is a possibility that they are incompatible with it for some reason.